app/models/access_passes_role.rb

```diff
@@ -0,0 +1,9 @@
+class AccessPassesRole < ApplicationRecord
+  belongs_to :access_pass
+  belongs_to :role
+
+  ALOWED_ROLES = [Role::ROLE_NAMES[:stock_fulfilment], Role::ROLE_NAMES[:order_fulfilment]]
+  ALOWED_ROLES_IDS = ALOWED_ROLES.map {|role| Role.find_by(name: role).try(:id) }
```

Comment:
This would benefit from dynamic assignment... i.e. not be set at boot-up time... imagine the case where a developer environment has no roles yet and then the roles are loaded in after the code has booted.

Suggest using a lamba for something like `Role.where(name: ALLOWED_ROLES).pluck(:id)`